srcpkgs/serviio/template

```diff
@@ -0,0 +1,49 @@
+# Template file for 'serviio'
+pkgname=serviio
+version=2.1
+revision=1
+archs="x86_64"
+depends="openjdk11 ffmpeg"
+short_desc="Free DLNA media server"
+maintainer="Domenico Panella <pandom79@gmail.com>"
+license="custom:serviio"
+homepage="http://www.serviio.org/"
+distfiles="http://download.serviio.org/releases/${pkgname}-${version}-linux.tar.gz"
+checksum=51748160b38e31ba54db1b11381d3d9d705852ed7757ad111e881327a16fb757
+restricted=yes
+
+do_install() {
+
+_SERVIIO_HOME=usr/share/java/$pkgname
+vmkdir ${_SERVIIO_HOME}
+
+vmkdir usr/bin
+vcopy bin ${_SERVIIO_HOME}
+ln -s /${_SERVIIO_HOME}/bin/${pkgname}-console.sh ${DESTDIR}/usr/bin/${pkgname}-console
+
+vcopy config ${_SERVIIO_HOME}
+
+vcopy lib ${_SERVIIO_HOME}
+
+vcopy library ${_SERVIIO_HOME}
+
+vcopy plugins ${_SERVIIO_HOME}
+
+vmkdir var/log
+ln -s /${_SERVIIO_HOME}/log ${DESTDIR}/var/log/${pkgname}
```

Comment:
Indentation
I prefer to reverse the link.

Reply:
If i make indentation, xlint returns always a error on the 17 line....( i don't know the reason ). All the lines have the same indentation!
To reverse th link, i need to create the folder empty that it is deleted at build time. 
How do I keep folders empty?

Reply:
i solved by make_dirs property

Reply:
i deleted serviio.png and serviio.service

Reply:
Will you let me know if anything else is needed?

Reply:
Make package able to build.

Reply:
Travis continues to give me an indentation error, xlint as well.
As you can see the 18 line is indented correctly.

Reply:
`srcpkgs/serviio/template:18: indent with tabs`

Reply:
*Tabs* refer to single character, sometimes presented as `\t`. Use it instead of spaces. You can configure your editor to use it in this file or copy from another template or ...